projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -80,23 +89,40 @@ export default function SubscribePanels() {
 			>
 				{ showNotices && (
 					<InspectorNotice>
-						{ createInterpolateElement(
-							sprintf(
-								/* translators: 1$s will be subscribers, %2$s will be social followers */
-								__( 'This post will reach <span>%1$s</span> and <span>%2$s</span>.', 'jetpack' ),
+						{ willShareToSocialMedia &&
+							createInterpolateElement(
 								sprintf(
-									/* translators: %s will be a number of subscribers */
-									_n( '%s subscriber', '%s subscribers', subscriberCount, 'jetpack' ),
-									numberFormat( subscriberCount )
+									/* translators: 1$s will be subscribers, %2$s will be social followers */
```

Comment:
Should we specify that this will be "a number of" subscribers and followers? I'm asking because as a translator, I think my first response to this comment would be to ask myself why "subscribers" and "social followers" are not part of the string, if that's what the placeholders are.

Reply:
Sure. I've updated this with 0ef2c83f0f9b4ac176b4e05b25e4f9fcf2754062